CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+This document outlines major changes between releases.
+
+## [Unreleased]
+
+New features:
+
+Behaviour changes:
+ * add generic Hash/Address parameters to `DBFT` service (#94)
+ * remove custom payloads implementation from default `DBFT` service configuration
+   (#94)
+
+Improvements:
+ * untie `dbft` module from `github.com/nspcc-dev/neo-go` dependency (#94)
+
```

Comment:
There also were dependency upgrades of various kinds.

Reply:
Fixed.